Workspace.swift

```diff
@@ -2,6 +2,15 @@ import WorkspaceConfiguration
 
 let configuration:WorkspaceConfiguration = .init()
 
+// ••••••• General •••••••
+
+// The project appears to only target Linux.
+// In any case, it fails to compile on macOS right now.
+configuration.supportedOperatingSystems.remove(.macOS)
```

Comment:
can you show me exactly where the macOS build is failing? harlan got it working in 2.0.1 and the intention was it should just work with minimal modifications

Reply:
It was having trouble with package config and I couldn’t get `zlib` set up right. When I only saw Linux mentioned on the read‐me and the Travis configuration I stopped trying. Try activating a macOS build on travis.

Reply:
Done.